app/code/Magento/Inventory/Test/Integration/Model/ResourceModel/SourceItem/DeleteMultipleTest.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model\ResourceModel\SourceItem;
```

Comment:
all the tests should be stored under the namespace 

`Magento\Inventory\Test\Unit|Integration\Model`

not mixed with code. Such tests would not be even run by Travis